doc/configuration.rst

```diff
@@ -0,0 +1,107 @@
+Configuration
+-------------
+
+All PuDB information is stored in a location specified by the `XDG Base
+Directory Specification
+<http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html>`_.
+Usually, it is ``~/.config/pudb``. The PuDB configuration is stored at
+``pudb.cfg`` file inside that directory. Therefore the whole path is usually
+``~/.config/pudb/pudb.cfg``.
+
+Following options are available to customise PuDB behaviour:
+
+shell
+*****
+
+This is the shell that will be used when you hit ``!``. Available choices:
+
+* internal
+* classic
+* ipython
+* bpython
+* ptpython
+
+theme
+*****
+
+PuDB UI theme. Select one available or use your own by setting ``custom_theme``.
+
+custom_theme
+************
+
+To use a custom theme, see example-theme.py in the pudb distribution. Enter
+the full path to a file like it in the box above. ``~`` will be expanded to
+your home directory. Note that a custom theme will not be applied until you
+close this dialog.
+
+
+line_numbers
+************
+
+Show or hide the line numbers in the source code pane.
+
+sidebar_width
```

Comment:
I agree with you. But before I update this PR I'd like to understand if has any chance to be merged.

@inducer are you strictly against of merging this PR or we can discuss and polish these changes to get them merged?